### DISCUSSION TITLE
[Virtual event] Chatwoot Connect is happening!

### DISCUSSION THREAD

**Discussion by Hricha-Shandily** [chatwoot/chatwoot#5043] [Releases & Announcements]
Hi, folks! 👋

You've all helped us make Chatwoot what it is. We have long been waiting for an opportunity to connect with you all.

And so, we are hosting our first-ever virtual event – **Chatwoot Connect**. 🎉

We'll be taking this opportunity to share our journey so far – how the founding team built an open-source project that raised $1.6M from YC; and what we have planned for the product going forward.

But, most importantly – we'll be hosting an AMA. 

Save the date: **26 July '22**. 

[**Tap here for more info and RSVPing**](https://www.chatwoot.com/chatwoot-connect/).

**Comment by jordan-brough** [comment]
We missed this! Do you have any recordings from the event or anything?

**Reply by pranavrajs** [reply]
Yes, it was live-streamed, and the link is here. https://www.youtube.com/watch?v=96snDChLfTM
